tools/codesigningtool/codesigningtool.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
```

Comment:
remove unused imports (absolute_import, print_function, base64, json)

Reply:
asbolute_import and print_function are cc'd from other scripts

Reply:
I'd rather keep them since we don't specify which python version to run them on

Reply:
gotcha, yeah, wasn't aware what they did.